### ISSUE TITLE
Macinnis-Ng_2004_2: Processing submission

### ISSUE THREAD

**Issue by ehwenk** [traitecoevo/austraits.build#440]
**Source**: Macinnis-Ng et al (2004) Variations in sensitivity to copper and zinc among three isolated populations of the seagrass, Zostera capricorni *Journal of experimental marine biology and ecology* http://doi.org/10.1016/j.jembe.2003.10.002
**Custodian**: Cate Macinnis-Ng
**Handled by**: Lizzy Wenk

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [ ] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [ ] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [ ] Metadata complete, builds into Austraits
- [ ] Report reviewed by AusTraits team
- [ ] Report sent to contributor ( label as `waiting on response` )
- [ ] Response received from contributor
- [ ] Files updated & report finalised
- [ ] Final checks ( all tests pass )
- [ ] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
